test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -21,6 +21,24 @@
 KUBE_ROOT="/home/kubernetes"
 KUBE_BINDIR="${KUBE_ROOT}/kubernetes/server/bin"
 
+function config-ip-firewall {
+  echo "Configuring IP firewall rules"
+  # The GCI image has host firewall which drop most inbound/forwarded packets.
```

Comment:
This is needed because GCI defaults to drop all inbound packet and will not forward packet either. Hence leaving the node useless.